articles/custom-domains/index.md

```diff
@@ -27,109 +27,75 @@ Currently, the following Auth0 features and flows support use of custom domains:
 * Emails (the links included in the emails will use your custom domain)
 * Database, Social, and Passwordless (One-Time Password variant only) connections
 
-::: note
-If you obtain an access token for the Management API using an authorization flow with your custom domain, you must call Management API using the custom domain (your token will be considered invalid otherwise).
-:::
-
-## Certificate Management
-
-Auth0 offers two options for certificate management: 
 
-* You can manage the certificates for your custom domains yourself
-* You can have Auth0 manage them on your behalf
+::: panel Token Issuance
+Auth0 issues tokens with the **iss** claim of whichever domain you used with the request. For example, if you used **https://northwind.auth0.com/authorize...** to obtain an access token, the **iss** claim of the token you receive will be **https://northwind.auth0.com/**. If you used your custom domain**https://login.northwind.com/authorize...**, the **iss** claim value will be **https://login.northwind.com/**.
 
-### Auth0-Managed Certificates
-
-Auth0 will manage the creation and renewal of the certificates for your custom domain. This is the simplest custom domains deployment option.
-
-### Self-Managed Certificates
-
-::: note
-Self-Managed Certificates are only available for enterprise customers.
+If you obtain an access token for the [Management API](/api/management/v2) using an authorization flow with your custom domain, you **must** call the Management API using the custom domain (your token will be considered invalid otherwise).
 :::
 
-You are responsible for managing your SSL/TLS certificates and configuring a reverse proxy to handle SSL termination and the forwarding of requests to Auth0.
+## Certificate Management
 
-Choose this option if:
+Auth0 offers two certificate management options.
 
-* You want to have more control of your certificates (such as choosing your own CA or certificate expiration)
-* You want to enable additional monitoring over your API calls to Auth0
+**Auth0-Managed Certificates** are those where Auth0 will manage the creation and renewal of the certificates for your custom domain. This is the simplest custom domains deployment option, and the scope of this document.
 
-If you choose to manage your own certificates, you'll need to use a reverse proxy configured using AWS CloudFront.
+The **Self-Managed Certificates** option is available for enterprise customers only. Choosing this option means that you are responsible for managing your SSL/TLS certificates and [configuring a reverse proxy to handle SSL termination and forwarding requests to Auth0](/custom-domains/self-managed-certificates).
```

Comment:
Maybe it's better to have the link on the `**Self-Managed Certificates** option ` text?